packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -738,7 +742,11 @@ export default defineComponent({
 				Object.assign(additionalProps, {
 					options: singleOrMultiSelectOptions,
 				})
-
+			if (component === 'KtFieldSingleSelect') {
+				Object.assign(additionalProps, {
+					actions: singleSelectActions,
```

Comment:
This is right now always active, implying that it’s a required field. You should add it as an option in the form field generator instead